src/components/IconComponent/styles.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+export const BackgroundIcon = styled.div`
+  position: absolute;
+  top: 0px;
+  right: 0px;
+`;
+
+export default BackgroundIcon;
```

Comment:
We don't need to export it as default, you can delete it. 

And delete all default exports from all _styled_ files

And add rule for _eslintrc.json_  `"import/prefer-default-export": "off",`